tests/ship_streamer.cpp

```diff
@@ -111,8 +113,9 @@ int main(int argc, char* argv[]) {
       stream.write(boost::asio::buffer(request_type.json_to_bin(request_sb.GetString(), [](){})));
       stream.read_message_max(0);
 
-      //      block_num, block_id
-      std::map<uint32_t, std::string> block_ids;
+      // Each block_num can have multiple block_ids since forks are possible
+      //       block_num,         block_id
+      std::map<uint32_t, std::set<std::string>> block_ids;
```

Comment:
why not `std::map<uint32_t, std::vector<std::string>>` since we never do lookups in the `set`?

Reply:
We do look up here: https://github.com/AntelopeIO/leap/pull/1299/files#diff-40f2ef69e45ff8a50b2d87a1f8515c6e6aeeb184ab3e4cc67eb8b62893e65f88R171